tests/unit_tests/test_tethys_apps/test_base/test_handoff.py

```diff
@@ -3,6 +3,11 @@
 from types import FunctionType
 from unittest import mock
 from tethys_sdk.testing import TethysTestCase
+from django.conf import settings
+
+prefix_to_path = ""
+if settings.PREFIX_TO_PATH is not None and len(settings.PREFIX_TO_PATH) != 0:
+    prefix_to_path = f"/{settings.PREFIX_TO_PATH}"
```

Comment:
Rather than getting settings in tests you should set them using `self.settings` context mananger or the `@override_settings` decorator. This will make the tests indepedendent of whatever settings are currently set in the devs environment. See: https://stackoverflow.com/questions/36611393/how-to-mock-django-settings-attribute-used-in-another-module

Reply:
I have a used also another method inside the Test classes to reload the URLs since they get formed with the extra settings.